### PR TITLE
DEVDOCS-6025 [update]: Orders V3, update order transactions

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-6025]

Relevant slack conversation https://bigcommerce.slack.com/archives/C96P5T3J4/p1723752403863899 
reposted https://bigcommerce.slack.com/archives/C9DGL6D8A/p1724174002476639 

## What changed?

* Update Orders V3 endpoints:
- Get Order Transactions
- Order (Batch) Metafields
- Order (Single) Metafields

## Release notes draft
N / A

## Anything else?
@bigcommerce/team-orders 

[DEVDOCS-6025]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-6025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -1924,13 +1928,16 @@ components:
...
                 A store credit model.
+              nullable: true
               properties:
                 remaining_balance:
                   description: |
                     Remaining balance of shopperʼs store credit.
                   type: number
                   format: float
                   example: 35.42
+            custom_provider_field_result:
```
Comment (bc-andreadao) [inline reference/orders.v3.yml:1939]: If someone could provide the schema and description for this, that would be great

Comment (WillemHoman) [reply reference/orders.v3.yml:1941]: @bigcommerce/team-payments 🙏

**Review by WillemHoman — APPROVED** [review]
Thanks @bc-andreadao.

FYI `/v3/orders/transactions` whilst under the orders namespace is maintained by the payments team as it just proxies data stored in the payments domain so you are best off pinging them for changes to that endpoint.

File: reference/orders.v3.yml (hunk 1 above)
Comment (bc-rajesh) [reply reference/orders.v3.yml:1941]: Here you go with the schema details.
```
custom_provider_field_result:
  type: object
  description: custom payment provider field result model.
  title: Custom Payment Provider Field
  nullable: true
  properties:
    receipt_number:
      type: string
      description: |
        The receipt number associated with the transaction.
      nullable: true
    authorization_code:
      type: string
      description: |
        Authorization code for the transaction.
      nullable: true
    fraud_response:
      type: string
      description: |
        The fraud response associated with the transaction.
      nullable: true
    amount_received:
      type: number
      format: float
      description: |
        The amount received for the transaction, divided by 100 to convert to the correct currency format.
      nullable: true
```

**Review by WillemHoman — APPROVED** [review]

**Review by bc-rajesh — APPROVED** [review]
lgtm

**Review by bc-traciporter — APPROVED** [review]
Approved

**Pull request merged**